smriprep/workflows/outputs.py

```diff
@@ -642,6 +644,34 @@ def init_anat_derivatives_wf(
         name="ds_surfs",
         run_without_submitting=True,
     )
+    # Sphere registrations
+    name_regs = pe.MapNode(
```

Comment:
seems like these are low memory nodes - probably should set mem_gb to something low